src/components/cylc/Tree.vue

```diff
@@ -36,6 +36,13 @@ export default {
       default: 0
     }
   },
+  watch: {
+    workflows: {
+      handler () {
+        this.clearCaches()
```

Comment:
A `watch` because this is a case where it is necessary to act when the element changes, and that we have no easy way with Vue default approaches like `v-if`, `data`, `computed`, etc.